.fttemplates/TS Functional Component/[FTName].test.tsx

```diff
@@ -3,7 +3,7 @@ import 'react-native';
 
 import { render } from '@testing-library/react-native';
 
-import {<FTName | capitalize >} from './[FTName]'
+import <FTName | capitalize> from './[FTName]';
```

Comment:
Since each folder has an index, styles and types file maybe this one should also be renamed to be just test.tsx?